auxiliary/auxiliary_test.go

```diff
@@ -0,0 +1,95 @@
+package auxiliary
+
+import "testing"
```

Comment:
If you want to improve your testing I recommend using https://github.com/stretchr/testify